qiita_pet/handlers/api_proxy.py

```diff
@@ -0,0 +1,117 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from __future__ import division
+
+# This is the only file in qiita_pet that should import from outside qiita_pet
```

Comment:
Are we sure we want to have this? Regrettably we have a few pieces of code that are "to be removed when X happens" and this compromises the usability, and the stability of those pieces of code and everything that gets built on top of them. If nobody sees this is as a possible problem, then lets leave as is.